tests/snuba/api/endpoints/test_organization_events_trace.py

```diff
@@ -6,122 +6,21 @@
 from django.urls import NoReverseMatch, reverse
 
 from sentry import options
-from sentry.issues.grouptype import NoiseConfig, PerformanceFileIOMainThreadGroupType
-from sentry.testutils.helpers import override_options
-from sentry.testutils.helpers.datetime import before_now, iso_format
+from sentry.issues.grouptype import PerformanceFileIOMainThreadGroupType
+from sentry.testutils.cases import TraceTestCase
+from sentry.testutils.helpers.datetime import before_now
 from sentry.utils.samples import load_data
 from tests.snuba.api.endpoints.test_organization_events import OrganizationEventsEndpointTestBase
 
 
-class OrganizationEventsTraceEndpointBase(OrganizationEventsEndpointTestBase):
+class OrganizationEventsTraceEndpointBase(OrganizationEventsEndpointTestBase, TraceTestCase):
```

Comment:
All the logic to support creating events and traces has been moved to `TraceTestCase`.

I will need this logic for my own work in another PR.

The Split view makes the changes on this file easier to review.